drivers/wifi/nrf700x/osal/fw_if/umac_if/inc/fw/host_rpu_sys_if.h

```diff
@@ -780,6 +780,16 @@ enum op_band {
 	BAND_24G
 };
 
+/**
+ * @brief This enum defines keep alive status.
+ */
+enum keep_alive_status {
```

Comment:
we should add a namespace `nrf_wifi_keep_alive_status`.

Nit, I guess in networking `keepalive` should be used.

Reply:
Its taken care